featurebyte/query_graph/sql/aggregator/forward.py

```diff
@@ -63,22 +63,26 @@ def _get_aggregation_subquery(self, specs: list[ForwardAggregateSpec]) -> LeftJo
         horizon_in_seconds = 0
         if spec.parameters.horizon:
             horizon_in_seconds = pd.Timedelta(spec.parameters.horizon).total_seconds()
+        end_point_expr_current: Expression = cast(
+            Expression,
+            parse_one(f"{point_in_time_epoch_expr.sql()}"),
```

Comment:
Usually there isn't a need for `parse_one()`. Here `parse_one(f"{point_in_time_epoch_expr.sql()}")` is equivalent to simply `point_in_time_epoch_expr`.